diff --git a/src/PluginWrapper.php b/src/PluginWrapper.php
index c2a2cbc..30327dd 100644
--- a/src/PluginWrapper.php
+++ b/src/PluginWrapper.php
@@ -187,10 +187,12 @@ class PluginWrapper
     {
         $return = array();
         foreach ($paths as $path) {
-            if (!$this->filesystem->isAbsolutePath($path)) {
-                $path = getcwd().'/'.$path;
+            if(!empty($path)) {
+                if (!$this->filesystem->isAbsolutePath($path)) {
+                    $path = getcwd().'/'.$path;
+                }
+                $return[] = $path;
             }
-            $return[] = $path;
         }
 
         return $return;
